src/server/pfs/server/testing/server_test.go

```diff
@@ -2611,6 +2639,7 @@ func TestPFS(suite *testing.T) {
 		require.Equal(t, BCommit, commitInfos[1].Commit)
 		require.Equal(t, CCommit, commitInfos[2].Commit)
 		require.Equal(t, DCommit, commitInfos[3].Commit)
+		<-done
```

Comment:
This seems like it is going to timeout in CI.  Maybe use a select with a context.

Reply:
I put the close in a defer, so we should be good to go if we reach this point.